.github/ISSUE_TEMPLATE/page-error.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Page Error
+about: Come across the error page while navigating our site? Let us know what went
+  wrong
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the error**
+What error message appeared? Was there a status code displayed?
+
+**To Reproduce**
+Steps to reproduce the error:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+What were you looking for when this error occurred?
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```

Comment:
I think it might make sense if we just put a general what device.

Reply:
Do you want the device/os/browser template or just ask for device and see what they give us?

Reply:
Just ask for device and see what they give us I think, I think people either know or don't know and we can always ask after.